nbs/03_matcher.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.                if graph_edge not in graph.edges:
</pre><p>instead of doing</p>

<pre class="ql-syntax" spellcheck="false">if not x:
 continue
if y:
  z()
</pre><p>its eaiser to do:</p>

<pre class="ql-syntax" spellcheck="false">if x and y:
 z()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Thank you! I was confused because I wanted to make sure that graph_edge is not in graph.edges before I call graph.edges[graph_edge].. Forgot that if the first condition in and is false it does not check the second one</p>